src/ILCompiler.ReadyToRun/src/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -1499,5 +1504,122 @@ private void embedGenericHandle(ref CORINFO_RESOLVED_TOKEN pResolvedToken, bool
             MethodDesc methodDesc = HandleToObject(handle);
             throw new RequiresRuntimeJitException("embedMethodHandle: " + methodDesc.ToString());
         }
+
+        private bool IsLayoutFixedInCurrentVersionBubble(TypeDesc type)
+        {
+            // Primitive types and enums have fixed layout
+            if (type.IsPrimitive || type.IsEnum)
+            {
+                return true;
+            }
+
+            if (!_compilation.NodeFactory.CompilationModuleGroup.ContainsType(type))
+            {
+                if (!type.IsValueType)
+                {
+                    // Eventually, we may respect the non-versionable attribute for reference types too. For now, we are going
+                    // to play is safe and ignore it.
```

Comment:
Nit: "play is safe"

Reply:
Hehe, I see I copy & pasted another typo from CoreCLR. Thanks for spotting, will fix!

Reply:
Fixed in 3rd commit.